src/greynirseq/utils/tokenize_splitter.py

```diff
@@ -56,3 +59,33 @@ def split_text(text):
             pg_data.append(sentence)
         data.append(pg_data)
     return data
+
+
+class SentenceSegmenter:
```

Comment:
en og is útgáfurnar hafa bókstaflega engan sameiginlegan kóða (fyrir utan self.lang = lang). Er einhver ástæða til að troða þeim í sama klasa? Í mesta lagi ætti það að vera sameiginlegur superclass.